plugins/catalog-backend/src/next/database/DefaultProcessingDatabase.ts

```diff
@@ -87,6 +87,7 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
       .update({
         processed_entity: JSON.stringify(processedEntity),
         cache: JSON.stringify(state),
+        // TODO: should this also be JSON stringfy?
```

Comment:
no, seems to be a string in the type already

Reply:
But what about multiple errors? Are they just stored as multiline strings? Seems a little strange to do that.

Reply:
yeah see below, this is expected to be an array of stringified errors currently - but that's not formalized yet, we just left it as a string and then we'll figure out the exact encoding of each array item later

Reply:
so probably remove the comment then